--- ./Makefile.orig	2009-09-07 13:42:30.000000000 +0200
+++ ./Makefile	2009-09-07 13:43:40.000000000 +0200
@@ -12,7 +12,7 @@
 BINDIR = $(PREFIX)/bin
 DATADIR = $(PREFIX)/share/$(PROGRAM)
 DOCDIR = $(PREFIX)/share/doc/$(PROGRAM)
-DESKTOP = $(DATADIR)/kornelix-$(PROGRAM).desktop
+DESKTOP = $(DESTDIR)$(DATADIR)/kornelix-$(PROGRAM).desktop
 MANDIR = $(PREFIX)/share/man/man1
 
 # replace CFLAGS and LFLAGS -O (oh) with -g for GDB debugging
